### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader:VoiceOver is unnecessarily announcing "loading image" after "External data source for new external table "combo box .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5973]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Tools Name: VoiceOver
MAS References: MAS1.3.1

**Steps to Reproduce:**

1. Install the app Azure Data Studio.
2. Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files.
3. Navigate till "External data source for new external table "combo box  .
4. Observe that screen reader is announcing "loading image" after "External data source for new external table "combo box  .

**Actual:**
Navigate till "External data source for new external table "combo box , after that screen reader is announcing "loading image"  then "you are currently on image ,inside a dialog. To exit this group ,press Control-Option-Shift-Up arrow"  .

**Expected:**
Screen Reader should announce the combo box and item list after "External data source for new external table "combo box .It should announce " SqlStoragePool , collapsed, button  ".

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader only users may get confused with image.

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[5973_[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3275433/5973_.Accessibility.A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This no longer seems to be an issue on the latest build from master
